distro/src/avail/Avail.avail/Foundation.avail/Atoms.avail

```diff
@@ -142,7 +142,7 @@ Public method "«Public»?_is a|an new⁇field⁇atom" is
 	atomName : string
 |
 	theAtom ::= atom for atomName;
-	Method atomName is [theAtom];
+	Macro atomName is [`theAtom`(atomName) → literal phrase];
```

Comment:
Oh, shoot.  It's targeting main instead of development, which is why it has all those extra commits.  I already merged a lot of those changes onto development, so this is noise.  I'll retarget the pull request, but I'll need to go through the rest of your comments first, so they won't be lost.